pkgs/tools/graphics/barcode/default.nix

```diff
@@ -13,10 +13,12 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "GNU barcode generator";
     maintainers = with maintainers; [ raskin ];
-    platforms = platforms.linux; # Maybe other non-darwin Unix
+    platforms = platforms.unix; # Maybe other non-darwin Unix
     downloadPage = "https://ftp.gnu.org/gnu/barcode/";
     updateWalker = true;
     homepage = "https://www.gnu.org/software/barcode/";
     license = licenses.gpl3;
   };
+  configureFlags = stdenv.lib.optional stdenv.isDarwin "ac_cv_func_calloc_0_nonnull=yes" ;
+  patches = stdenv.lib.optional stdenv.isDarwin [ ./patches/darwin.patch ];
```

Comment:
Please move this above meta under hardening disabled. The patch should be directly under src.

Reply:
The patch should be sent upstream

Reply:
Please move those two lines (`configureFlags = ...` and `patches = ...`) above the line `meta = ...`. `meta` should be the last argument to `mkDerivation` for readability.